### PR TITLE
Assignment2 @PlantBrush

### PR DESCRIPTION
**Pull request by NazariyPogo**
URL: https://lemon-ground-09e5fcf1e.1.azurestaticapps.net/

### CODE REVIEW

**Comment by sammyboy2000** [comment]
Good work on part 1. Have you considered placing the button elsewhere, like on the bottom of the keyboard, or between the gameboard and keyboard?
Part 2 and 3 look good.
Part 4 looks good except for a lack of gradients.
You've got a link to your website, so there's part 5.

**Review by xiokd — COMMENTED** [review]
The web app is deployed to Azure and works well. Almost every specification has been completed. I have left a comment where a possible revision can be made. Overall, everything looks good. Nicely done!

**Comment by schuywalker** [comment]
I will review

File: wordle-web/components/burger-menu.vue [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<template>
+  <v-card>
+    <v-menu offset-y open-on-hover>
```
Comment (mmwoodfo) [inline wordle-web/components/burger-menu.vue:3]: Nice job with this menu. You don't _have_ to wrap it in a `v-card`

File: wordle-web/components/game-board.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <v-card width="450">
-    <v-container>
+    <v-container id="gameboard_container" class="fancy_gradient">
```
Comment (mmwoodfo) [inline wordle-web/components/game-board.vue:3]: This is a very fancy_gradient. I like it, makes the game very trippy to play.

**Review by mmwoodfo — COMMENTED** [review]
Good job on the assignment. I like that the number updates as you type words! :)
My only suggestion would be to close the dialog once you select a word.

**Pull request closed without merging**